test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -1763,10 +1728,23 @@ private class EmptyModel
         {
         }
 
+        private class BindingOptionalProperty
+        {
+            [BindingBehavior(BindingBehavior.Optional)]
+            public int ValueTypeRequired { get; set; }
+        }
+
+        private class NullableValueTypeProperty
+        {
+            [BindingBehavior(BindingBehavior.Optional)]
+            public int? NullableValueType { get; set; }
+        }
+
         private class Person
```

Comment:
This was used in plenty of other tests, so I updated the model to behave like those tests expect it to.